src/catalua.h

```diff
@@ -29,6 +110,12 @@ int call_lua( std::string tocall );
 int lua_mapgen( map *m, const oter_id &terrain_type, const mapgendata &md, int t, float d,
                 const std::string &scr );
 
+/**
+ * Execute a callback that can be overridden by all mods,
+ * storing provided callback arguments in _G.
+ */
+void lua_callback( const char *callback_name, CallbackArgumentContainer callback_args );
```

Comment:
Why does this function take a *copy* of the `CallbackArgumentContainer` object and not just a const reference?